kalite/distributed/data/screenshots.json

```diff
@@ -0,0 +1,84 @@
+[
+    {
+        "users": ["guest"],
+        "slug": "home",
+        "start_url": "/",
+        "inputs": [],
+        "pages": [5],
+        "notes": "Does this annotation work?",
```

Comment:
We'd also want some documentation on how to use this. Not sure if we have decided where we want developer documentation to be yet though.